src/ElevatorButton.java

```diff
@@ -0,0 +1,3 @@
+public class ElevatorButton {
```

Comment:
shouldn't the elevator button have an instance variable for the floor number and if the button is on/off?